src/Checkbox/CheckboxText.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+
+import * as React from 'react';
+import { Text, View, Platform } from 'react-native';
+import StyleSheet from '../PlatformStyleSheet';
+import theme, { parsePxValue } from './styles';
+
+type Props = {|
+  +label?: React.Node,
+  +checked?: boolean,
+  +info?: React.Node,
+|};
+
+const styles = StyleSheet.create({
+  view: {
+    display: 'flex',
+    flex: 1,
+    flexDirection: 'column',
+    marginStart: parsePxValue(theme.orbit.spaceXSmall),
+  },
+  label: {
+    fontFamily: Platform.OS === 'web' ? theme.orbit.fontFamily : 'Roboto',
```

Comment:
I believe this is a very general "problem" to solve, so we should maybe extract it into a helper function?

Reply:
You're right, so no need to refactor this now.